core-bundle/src/DependencyInjection/Filesystem/FilesystemConfiguration.php

```diff
@@ -59,8 +59,9 @@ public function addVirtualFilesystem(string $name, string $prefix, bool $readonl
         $definition->setFactory(new Reference('contao.filesystem.virtual_factory'));
         $definition->addTag('contao.virtual_filesystem', ['name' => $name, 'prefix' => $prefix]);
 
+        $aliasName = str_replace('_', '', ucwords($name, '_'));
```

Comment:
```suggestion
        $aliasName = str_replace('_', '', lcfirst(ucwords($name, '_')));
```
To get `$newsMediaStorage` instead of `$NewsMediaStorage`